websites/H/Hako/dist/metadata.json

```diff
@@ -0,0 +1,27 @@
+{
+  "author": {
+    "name": "kinue",
+    "id": "634246091293851649"
+  },
+  "service": "Hako",
+  "description": {
+    "en": "Reading vietnamese translated light/web novels."
```

Comment:
if you support vietnamese in presence add vietnamese desc too. Also you should do it as description not like you're doing this thing. Doesn't this site have some description or whatever? If not change it to ``Read vietnamese translated light/web novels online.``